client/src/components/componentsUser/Login.js

```diff
@@ -13,34 +16,51 @@ export default class CreateAccount extends Component{
   }
 
   handleChange = (e) => {
-    // console.log(e.target.id, e.target.value);
+
     this.setState({
       [e.target.id]: e.target.value
     });
   }
 
   handleSubmit = (e) => {
-    //console.log(this.state.username);
+    //console.log( this.state.username);
     this.props.login(this.state.username, this.state.password);
+    localStorage.setItem('username', this.state.username);
+    localStorage.setItem('password', this.state.password);
     e.preventDefault();
   }
 
-  render () {
-    return (
-         <div>
+    render() {
+      let logout = () => {
+        f.signout()
+      }
+      let ingresa = () => {
+        localStorage.setItem('username', this.state.username);
+        localStorage.setItem('password', this.state.password);
```

Comment:
I don't see why you need to store this into localStorage here. You should call `authenticate` with this state and that's it.

the call should looks like `authorization.authenticate(this.state.username, this.state.password)` and into `authenticate` function if you are correctly authenticated then set the credential into localStorage there

(btw: change `f` to `authorization`)